interface/src/app/node/enriched_genesets.tpl.html

```diff
@@ -0,0 +1,21 @@
+<h4>
+  <b>Enriched gene sets: </b>
+</h4>
+
+<table id="enrichment-table" class="table table-striped table-bordered table-condensed">
+  <thead>
+    <tr>
+      <td>Pathways/Processes/Diseases</td>
```

Comment:
Yes, **th** is recommended inside **thead**. You will see the difference when you make the change.